code/game/gamemodes/game_mode.dm

```diff
@@ -164,6 +164,7 @@
 		feedback_set("escaped_on_pod_5",escaped_on_pod_5)
 
 	send2mainirc("A round of [src.name] has ended - [surviving_total] survivors, [ghosts] ghosts.")
+	send2maindiscord("A round of **[name]** has ended - **[surviving_total]** survivors, **[ghosts]** ghosts.")
```

Comment:
You may as well fold these into one proc for the future so people don't leave one out - it won't kill IRC to get the extra formatting.